Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/StudentSearch.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+public class StudentSearch {
+    public static void main(String[] args) {
+        int[] studentIDs = {12345, 67890, 54321, 98765, 24680};
+
+        // Вводимо номер залікової книжки
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введіть номер залікової книжки: ");
+        int gradebook = scanner.nextInt();
+
+        // Перевіряємо номер залікової книжки
+        for (int i: studentIDs) {
```

Comment:
Usually in for-each variation we name the temp variable so it represents what's inside it during the iteration

`int id : studentIDs`

Not a mistake, but this will help to improve the readability